supplierinfo_product_breakdown/tests/test_product_supplierinfo_component.py

```diff
@@ -0,0 +1,30 @@
+from odoo import exceptions
+from odoo.tests import Form, tagged
+
+from .common import SupplierInfoCommon
+
+
+@tagged("post_install", "-at_install")
+class TestProductSupplierInfoComponent(SupplierInfoCommon):
```

Comment:
Please provide test scenario. What are you trying to test exactly?

Reply:
Resolved